testing/fixtures/comment-doc-on-hover.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+
+# this is a comment
+# describing the function
+# hello_world
+# this function takes two arguments
+hello_world() {
+    echo "hello world to: $1 and $2"
+}
+
+
+
+# if the user hovers above the below hello_world invocation
+# they should see the comment doc string in a tooltip
+# containing the lines 4 - 7 above
+
+hello_world "bob" "sally"
+
+
+
+# doc for func_one
+func_one() {
+    echo "func_one"
+}
+
+# doc for func_two
+# has two lines
+func_two() {
+    echo "func_two"
+}
+
+
+# this is not included
+
+# doc for func_three
+func_three() {
```

Comment:
Let us add another example that doesn't have any immediate documentation. And we could actually make that a variable instead to show that the function also works for variables (not only functions).